.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+
+Thank you for your Pull Request!  We have developed this task checklist from the [Development Process Guide](https://roche-gsk.github.io/admiral/articles/development_process.html) to help with the final steps of the process.  Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   
+
+Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request.  This checklist is part of the Github Action workflows and the Pull Request will not be merged into the devel branch until you have checked off each task.
+
+- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
+- [ ] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://roche-gsk.github.io/admiral/articles/unit_test_guidance.html)
```

Comment:
minor point but in the dev process vignette i used link https://roche-gsk.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral- here to go directly to the section they'll most need (although hopefully they read the whole thing if brand new)

Reply:
I updated the link.